make.py

```diff
@@ -0,0 +1,249 @@
+"""
+Helper script to build langkit and its dependencies.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from contextlib import contextmanager
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+
+from langkit.utils import add_search_path
+
+
+parser = argparse.ArgumentParser(
+    description='Helper script to build langkit and its dependencies'
+)
+parser.add_argument('--build-mode', default='dev')
+
+subparsers = parser.add_subparsers()
+
+build_parser = subparsers.add_parser('build')
+build_parser.set_defaults(command='build')
+
+setenv_parser = subparsers.add_parser('setenv')
+setenv_parser.set_defaults(command='setenv')
+
+install_parser = subparsers.add_parser('install')
+install_parser.set_defaults(command='install')
+install_parser.add_argument('prefix', type=str,
+                            help='Where to install Langkit dependencies')
+install_parser.add_argument('--gnat-install-dir', type=str, default=None,
+                            help='Specify where GNAT is installed. If not'
+                                 'provided, assume it is installed where '
```

Comment:
A space is missing between "not" and "provided". We never decided it was as a rule, yet I advise to put the spaces at the beginning of the next line to make it clear when it’s missing.